cli/cli.go

```diff
@@ -115,7 +115,12 @@ func InitCtx(cfg *Config, authn bool) *HttpContext {
 	if cfg.IsTenantInHost() {
 		basePath = "/SAAS" + vidmBasePath
 	}
-	ctx := NewHttpContext(cfg.Log, cfg.Option(HostOption), basePath, vidmBaseMediaType)
+	// insecureSkipVerify not nil when passed as a flag on the command-line which takes precedence over the options
+	skipVerify := cfg.Option(InsecureSkipVerifyOption) == "yes" || cfg.Option(InsecureSkipVerifyOption) == "true" // TODO support boolean values in the target options ?
```

Comment:
I would like to refactor the config's options from a `map[string]string` to a `map[string]interface{}` in order to natively support booleans.

I have refrained from doing this at the moment in order to minimize the code changes. 
Let me know if you want me to do that change as well.

Reply:
That would be nice to support boolean indeed.
Would you mind adding a unit test for this functionality as well?

Thanks for contributing!

Reply:
Ok, I'll refactor the map and will add a testunit.
Cheers!

Reply:
@fstrudel I added some unit tests that starts a TLS server and made sure insecure-skip-verify is needed and fixes the problem.
I also refactored the map of options as discussed.

Thanks for your attention!